include/open62541/client.h

```diff
@@ -186,6 +187,22 @@ typedef struct {
     size_t sessionLocaleIdsSize;
 } UA_ClientConfig;
 
+/**
+ * @brief It makes a partial deep copy of the clientconfig. It makes the shallow
+ * copies of the plugin (logger, eventloop, securitypolicy).
+ */
+UA_EXPORT UA_StatusCode
+UA_ClientConfig_copy(UA_ClientConfig const *src, UA_ClientConfig *dst);
+
+/**
+ * @brief It cleans the client config and frees the pointer.
+ */
+UA_EXPORT void
+UA_ClientConfig_delete(UA_ClientConfig *config);
+
+
+UA_EXPORT void
+UA_ClientConfig_clear(UA_ClientConfig *config);
```

Comment:
Document for _clear that it **will delete the plugins** whereas _copy makes a shallow copy of the plugins.